src/jobs/recommendation.service.ts

```
@@ -0,0 +1,625 @@
+
+  /**
+   * Generate personalized job recommendations for a user
+   */
+  async generateRecommendations(
+    userId: string,
+    options: GetRecommendationsDto = {},
+  ): Promise<RecommendationResponseDto[]> {
+    const { limit = 20, offset = 0, preferences } = options;
+
+    // Get user preferences (either from request or stored)
+    const userPreferences = preferences || await this.getUserPreferences(userId);
```

ESLint

[error] Replace `·preferences·||·await·this.getUserPreferences(userId` with `⏎······preferences·||·(await·this.getUserPreferences(userId)`

(prettier/prettier)